### ISSUE TITLE
[BUG] CPU Usage using a wrong format or not working correctly on windows 11 with live 11.3.35

### ISSUE THREAD

**Issue by TheDogHusky** [Serena1432/DAWRPC#13]
Here is a screenshot: https://safe.classydev.fr/brave_0_J_Iu_Dz9_X8_R-YQPTwxB8xFmBZaNfGsO7cTOk0O521EKj.png

Why is it a 0.xx% of CPU?
If it is what ableton takes on the cpu, it is always wrong as ableton takes at least 4% of the CPU.
If it is the total CPU usage, why is it not 43% instead of 0.43% (format)?

I'd like to know what it means and if it's normal.

**Comment by Serena1432** [comment]
Sorry to disappoint you, but that could be Windows 11's fault.

Some Windows 11 installations usually returns wrong CPU usage in processes other than Task Manager. My previous Windows 11 installation used to encounter that same error when I was making a Discord BOT displaying my computer's CPU usage.
After investigating, I found that it was because of "performance counters" returning wrong value, but I don't know exactly why and how to fix that issue. The only solution back then when I encountered it was downgrading to Windows 10, and the issue was fixed.

And also, that wasn't "displaying 0.43 instead of 43%". In other Windows 10- computers that I've tested, the software always returned the correct CPU usage.

I'm afraid that I'm unable to fix the issue, since it isn't DAWRPC's fault. If you find a solution about "performance counters" issue that I described above, I will be very glad as I also used to encounter it!
